dgbpy/bokeh_apps/trainui/main.py

```diff
@@ -279,7 +279,7 @@ def mlchgCB( attrnm, old, new):
   def updateUI():
     nonlocal platformfld
     selplatform = platformfld.value
-    if info[dgbkeys.learntypedictstr] == dgbkeys.seisimgtoimgtypestr:
+    if info[dgbkeys.learntypedictstr] == dgbkeys.seisimgtoimgtypestr or info[dgbkeys.learntypedictstr] == dgbkeys.seisclasstypestr:
```

Comment:
info should not be manually parsed, use:
dgbpy.hdf5.‎isImg2Img( info ) or dgbpy.hdf5.‎‎isSeisClass( info )